### PR TITLE
Fix embeds according to N12 comments

### PR DESCRIPTION
**Pull request by zhiln**
This PR includes fixes for FAcebook, AP, Vimeo, Youtube

### CODE REVIEW

File: src/resolvers/contentful/facebook.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default (node) =>{
   } else {
     return `
       <div class="${config.embed.class} ${config.embed.facebook.class}">
-        <div class="fb-${type}" data-href="${url}"></div>
+        <div class="fb-${type}" data-href="${url}" data-width="500"></div>
```
Comment (ignatospadov) [inline src/resolvers/contentful/facebook.js]: Is that the max width we need?

Comment (zhiln) [reply src/resolvers/contentful/facebook.js]: that was just an experiment, I rolled it back few minutes ago

Comment (zhiln) [reply src/resolvers/contentful/facebook.js]: right now it uses fluid width, takes 100% of parent

**Review by ignatospadov — APPROVED** [review]
LGTM

**Pull request merged**